tests/simple_shear_output_the_mobility.prm

```diff
@@ -0,0 +1,133 @@
+# This parameter file is a copy of simple_shear.prm but with the added postprocessor
+# "mobility statistics" following Lourenco et al., 2020.
+# Mobility = surface rms velocity / whole domain rms velocity.
+# This value is ~1.73 in this test.
```

Comment:
Don't be modest here, the test is incredibly close to the analytical result of sqrt(3) now (looking at the statistics file). I would write this as follows (also to emphasize that his was tested against an analytical result):
```suggestion
# This result of ~1.73205 in this test is very close to the analytical solution of sqrt(3).
```